airflow/providers/amazon/aws/auth_manager/aws_auth_manager.py

```diff
@@ -142,7 +166,12 @@ def is_authorized_view(
         access_view: AccessView,
         user: BaseUser | None = None,
     ) -> bool:
-        return self.is_logged_in()
+        return self.avp_facade.is_authorized(
+            method="GET",
+            entity_type=AvpEntities.VIEW,
+            user=user or self.get_user(),
+            entity_id=access_view.value,
```

Comment:
Why don't we have a check for this value and default to None like the others?

Reply:
`access_view` is a required field

Reply:
Why is this one required but not the others? I think maybe the answer to my other comment might answer this as well.

Reply:
The other are optional because when you do authorization request, you might not need to specify a resource ID. Example: "Does the user have permission to create a Variable". There is no resource ID here. Here this is not really a resource ID but more a specific of which view the user is trying to access. We must specify which view the user is trying to access